README.md

```diff
@@ -100,12 +101,12 @@ The plugin artifacts are published to Maven Central and Github. To install a pre
 From Github:
 
 ```
-./bin/elasticsearch-plugin install https://github.com/Automattic/elasticsearch-statsd-plugin/releases/download/7.9.3.0/elasticsearch-statsd-7.9.3.0.zip
+./bin/elasticsearch-plugin install https://github.com/Automattic/elasticsearch-statsd-plugin/releases/download/7.8.1.0/elasticsearch-statsd-7.8.1.0.zip
 ```
 
 From Maven Central:
 ```
-./bin/elasticsearch-plugin install http://repo1.maven.org/maven2/com/automattic/elasticsearch-statsd/7.9.3.0/elasticsearch-statsd-7.9.3.0.zip
+./bin/elasticsearch-plugin install http://repo1.maven.org/maven2/com/automattic/elasticsearch-statsd/7.8.1.0/elasticsearch-statsd-7.8.1.0.zip
```

Comment:
is there a reason the readme references 7.8.1 rather than 7.10.2?

Reply:
umm, no, not really, It was in the PR already,  I just bumped pom.xml. Updating shortly.